app/assets/scss/_search-page.scss

```diff
@@ -1,94 +1,60 @@
-#search-page-heading .page-heading {
-  margin-bottom: 10px;
+.app-js-hidden {
+  display: none;
+  visibility: hidden;
 }
 
-.search-page-filters {
-
-  $lot-filters__gutter: 24px;
+/**
+ * Lot/category filter list
+ */
+.app-lot-filter {
 
-  .lot-filters {
+  a {
+    text-decoration: none;
+  }
```

Comment:
Is there any reason to remove the link underlines? I'm not certain but I'd worry about falling foul of [WCAG 2.1 failure 73](https://www.w3.org/WAI/WCAG21/Techniques/failures/F73)

Reply:
Ah yes, thank you - I was going to bring this up. I kept this to match what the current list looks like, but I want to jettison and make the links underlined for precisely that reason.

Reply:
Jettison away 👾